pangeo/requirements.yaml

```
@@ -1,7 +1,7 @@
 # requirements.yaml
 dependencies:
 - name: jupyterhub
-  version: "0.9.0-alpha.1.085.6ccbe82"
+  version: "0.9.0-beta.2.000.bed779e"
```

```suggestion
  version: "0.9.0-beta.2"
```

Both these are exactly the same, perhaps using beta.2 only could help avoid suggesting otherwise.

I'll probably update our CI/CD system to avoid releasing two when we push a git tag like beta.2!

---

Okay sounds good, I have removed the commit hash from the version.